packages/contentful-app-extensions/field-as-updated-at/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "isolatedModules": true,
+    "jsx": "react",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "noFallthroughCasesInSwitch": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": true,
+
+    "target": "es5",
```

Comment:
Is this the right target for these extensions? That's from 2009?

Reply:
I've changed to es2015 now https://github.com/yldio/asap-hub/pull/2922/commits/3c135c50ffd0cf3d0bde6ac4e29685fefa23c013